modules/402-ingress-nginx/openapi/values.yaml

```diff
@@ -230,13 +230,17 @@ properties:
                 additionalLogFields:
                   type: object
                   additionalProperties: true
+                additionalImagePullSecrets:
```

Comment:
To discuss.
* Theese aren't secrets, theese are secretNames.
* What if user didn't deploy the secret to d8-ingress-nginx namespace?
* Why don't let user configure dockerConfigJSON?